### DISCUSSION TITLE
Just a thank you and a very small suggestion

### DISCUSSION THREAD

**Discussion by shaft17** [DanielRendox/RoutineTracker#12] [Minor Enhancements & Suggestions]
Hey Daniel, just wanted to leave a thank you for this app. I've found it through a Youtube video from HowToMen.
I've read your plans for the future and I'm looking forward to it. When you're working on editing a task/habit, it would be nice to change colors of the task too, so you can group them as a theme. (green for cooking, red for sports, yo get it)
Thank you!

**Comment by DanielRendox** [comment]
Hey, thanks for the suggestion! I thought about that as well. 

Would you want colors on a specific task/habit details page change too? So tints of different UI elements designated to the specific activity would adapt to the selected color.

**Reply by shaft17** [reply]
Thanks for replying!
Well, in my case I wouldn't prefer colored detail pages. If UI elements change color every time, this could be confusing. Maybe someone will find that useful, but this is unneccessary in my eyes.

No, just the possibility to add more distinguishable items on the main page. Either by colored text or a nice tinted background as you have in the row of dates.
But - please make it optional :D If there are Nothing Phone users, I'm sure they'd like to stick to plain text without any colored background/text at all.

I'm looking forward to your development!
